devito/ops/transformer.py

```diff
@@ -271,7 +271,7 @@ def create_ops_arg(p, accessible_origin, name_to_ops_dat, par_to_ops_stencil):
     elements_per_point = 1
     dtype = Literal('"%s"' % dtype_to_cstr(p.dtype))
 
-    if p.is_Constant:
+    if p.is_Scalar:
```

Comment:
why?

Reply:
an `ops_par_loop` method receives either `ops_arg_dat` (arrays) or `ops_arg_gbl` (numbers)
these types are represented by either a `RawAccessible` or a `OpsAccessible`.
both inherit from basic.Symbol. thus it seems it illogical that those should be marked as a `Constant` (https://github.com/opesci/devito/blob/master/devito/types/basic.py#L168)